playbooks/roles/network_info/tasks/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Read network info if present
   slurp:
-    src: &networkinfo /etc/dev-install-network-info.yaml
+    src: &networkinfo /tmp/dev-install-network-info.yaml
```

Comment:
~~Would prefer `/var/tmp`, as `/tmp` is frequently on tmpfs and not preserved between reboots:~~

~~https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s15.html~~

Although reading that, even `/var/tmp` isn't right. I think we should write this to `/var/lib/`:

https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s08.html

We never want anything to clean it up.